cmd/influxd/inspect/verify_tombstone.go

```diff
@@ -0,0 +1,137 @@
+package inspect
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/influxdata/influxdb/v2/tsdb/engine/tsm1"
+	"github.com/spf13/cobra"
+)
+
+type argsTomb struct {
+	dir string
+	v   bool
+	vv  bool
+	vvv bool
+}
+
+type verifierTomb struct {
+	path      string
+	verbosity int
+	files     []string
+	f         string
+}
+
+const (
+	quiet = iota
+	verbose
+	veryVerbose
+	veryVeryVerbose
```

Comment:
I don't see this being used anywhere, it seems like `veryVerbose` is the most verbose logging we have now.